src/components/JoinStaffCard/index.js

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './joinStaffCard.scss';
+
+const CancelButton = () => {
+	return(
+		<a className="cancel-button" >
+    		<span className="icon has-text-grey">
+			  <i className="fas fa-minus-circle"></i>
+			</span>
+    	</a>
+	);
+}
+
+export const JoinStaffCard = ({name, phoneNumber, email, tickets, tenants, admin}) => {
+	
+	const InfoLabel = ({type}) => {
+		if (type === "tickets"){
+			return (
+				<div className="info-font">
+			    	<p className="number-font" >{tickets}</p> Open Tickets
+			    </div>
+			);
+		} else if (type === "tenants") {
+			return (
+				<div className="info-font">
+			    	<p className="number-font" >{tenants}</p> Tenants
+			    </div>
+			);
+		}
+	}
+
+	const AdminLabel = () => {
+		return (
+			<div className="admin-label"> Admin </div>
+		);
+	}
+
+	const tinyLinkSpacing = !(tickets && tenants) ? "tiny-link" : "tiny-link-info";
+
+	return (
+		<>
+			<div className="card card-length">
```

Comment:
Looks great so far! I noticed though there's some extra spacing in the cards that is pushing the "Reassign" text out of the card boundaries. Let's fix up some of the styling so it more matches the mockup:
![image](https://user-images.githubusercontent.com/8600792/81774960-a6108480-94a0-11ea-8fb0-14501cb78bb8.png)


This was the run-off I was seeing on Chrome:
![image](https://user-images.githubusercontent.com/8600792/81774995-b7599100-94a0-11ea-99b9-d6b9274e6f02.png)